packages/transform-adoc/package.json

```diff
@@ -33,9 +33,8 @@
   "devDependencies": {
```

Comment:
Do we need to update the react-core version in this package?

Reply:
Not sure, but probably wouldn't hurt?